### DISCUSSION TITLE
q-splitter limits not working

### DISCUSSION THREAD

**Discussion by asudevprj** [quasarframework/quasar#15127] [General - Components / Directives / etc]
I am trying to use the q-splitter on a **mobile** screen. The screen is splitted horizontally (before / after) and should always use the whole mobile screen hight. If I set the 'after' template to its maximum limit [80, 100]  programmatically (80) and then scroll the 'before' screen down, the splitter limits are changed, the after screen hight is increased. It seems that the 'after' screen hight  is increased with the amount of the app header hight. No footer is used.
 How do I prefent, that the 'after' screen changes its height?

The code:

<template>
  <q-layout view="hHh lpR fFf" >
    <q-header elevated>
      <q-toolbar>
        <q-toolbar-title>
          Quasar App
        </q-toolbar-title>
      </q-toolbar>
    </q-header>

    <q-page-container >
      <q-page class=" flex ">
        <div >
          <q-splitter
            v-model="splitterModel"
            horizontal
            :limits="[80, 100]"
            class="full-width"
            style="height: 100vh"
          >
            <template v-slot:before>
              <div class=" bg-purple " >
                  <div class="text-h4 q-mb-md">Before</div>
                  <div v-for="n in 20" :key="n" class="q-my-md">{{ n }}. Lorem ipsum dolor sit, amet consectetur adipisicing elit. Quis praesentium cumque magnam odio iure quidem, quod illum numquam possimus obcaecati commodi minima assumenda consectetur culpa fuga nulla ullam. In, libero.</div>
              </div>
            </template>

            <template v-slot:after>
              <div class="bg-green ">
                  <div class="text-h4 q-mb-md">After</div>
                  <div v-for="n in 20" :key="n" class="q-my-md">{{ n }}. Lorem ipsum dolor sit, amet consectetur adipisicing elit. Quis praesentium cumque magnam odio iure quidem, quod illum numquam possimus obcaecati commodi minima assumenda consectetur culpa fuga nulla ullam. In, libero.</div>
              </div>
            </template>
          </q-splitter>
        </div>
      </q-page>

    </q-page-container>
  </q-layout>
</template>

<script setup>
import { ref } from 'vue'

const splitterModel = ref(80)

</script>

**Comment by pdanpdan** [comment]
Please make a codepen

**Reply by pdanpdan** [reply]
you set a height `style="height: 100vh"` that can be (most probably is) larger than the available space in page

**Comment by asudevprj** [comment]
Unfortunately I am not able to produce a codepen with the behaviour I am experiencing in my browser.

Further info to reproduce:
After scrolling the 'before' screen and then draging the 'before' screen upwards, the whole screen moves upwards and thereby increases the 'after' screen hight. It seems as it the splitter does not take into account the height of the header (or unused footer?) ?

**Reply by asudevprj** [reply]
Yes, you are right! 100vh is the problem! Thanks a lot, I missed that.

How to set the hight so that on every mobile device the whole screen hight is used?
